<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolygonMainWidget</class>
 <widget class="QWidget" name="PolygonMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>ScanLine</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="gridWidth">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>110</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>577</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Width</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>200</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="timeDisplay">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>479</width>
     <height>479</height>
    </rect>
   </property>
   <widget class="ScanLineFillWidget" name="page_0"/>
   <widget class="BoundaryFillWidget" name="page_1"/>
   <widget class="FloodFillWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="drawPolygonButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Draw Polygon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seedButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Select Seed</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScanLineFillWidget</class>
   <extends>QWidget</extends>
   <header>polygons/scanlinefillwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryFillWidget</class>
   <extends>QWidget</extends>
   <header>polygons/boundaryfillwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FloodFillWidget</class>
   <extends>QWidget</extends>
   <header>polygons/floodfillwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
